source/pom.xml

```diff
@@ -72,7 +72,7 @@
         <jcommon.version>1.0.16</jcommon.version>
```

Comment:
The inline comments are helpful; however, a method docstring would provide a high-level overview.